src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -1839,17 +1855,3 @@ services:
     public: true
     tags:
       - { name: form.type }
-
-  form.type.common.zone_choice:
```

Comment:
Why remove those forms? Is it maybe a typo or a mistake after a rebase (since I believe you worked on these types recently in another PR maybe some contents were mixed)?

It doesn't seem related to the PR anyway so it seems strange.